CMakeLists.txt

```diff
@@ -239,6 +242,9 @@ if(GEN_CPP_BINDINGS)
         add_subdirectory(swig)
     endif()
 endif()
+if(GEN_LANGUAGE_BINDINGS)
+    add_subdirectory(swig)
```

Comment:
This option should be global and duplicating code avoided. But I can fix this myself.

Reply:
I'm not really sure what you mean by that if you can elaborate I can do it, but it's fine if you do it.